src/tools/ip/client/CMakeLists.txt

```diff
@@ -1,17 +1,11 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${QUIC_CXX_FLAGS}")
-
-include_directories(${PROJECT_SOURCE_DIR}/src/tools/ip)
-
+# This does not use add_quic_tool because it doesn't use platform
```

Comment:
Does it break things to include platform though? Shouldn't the linker just ignore it?

Reply:
It doesn't technically break things. So we could add that.